README.md

```diff
@@ -1,19 +1,21 @@
 [![CI](https://github.com/postfinance/kubenurse/actions/workflows/release.yml/badge.svg)](https://github.com/postfinance/kubenurse/actions/workflows/release.yml)
 [![Coverage Status](https://coveralls.io/repos/github/postfinance/kubenurse/badge.svg?branch=master)](https://coveralls.io/github/postfinance/kubenurse?branch=master)
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/postfinance/kubenurse)
 
 # Kubenurse
-kubenurse is a little service that monitors all network connections in a kubernetes
-cluster and exports the taken metrics as prometheus endpoint.
+kubenurse is a little service that monitors all network connections in a Kubernetes
+cluster. Kubenurse measures request durations, records errors and exports the taken metrics in Prometheus format.
 
 ## Deployment
-You can get the Docker Image from [Docker Hub](https://hub.docker.com/r/postfinance/kubenurse/).
+You can get the Docker image from [Docker Hub](https://hub.docker.com/r/postfinance/kubenurse/).
 The [examples](https://github.com/postfinance/kubenurse/tree/master/examples) directory
-contains yamls that deploy the kubenurse to the kube-system namespace.
+contains YAMLs which can be used to deploy the kubenurse to the kube-system namespace of your cluster.
```

Comment:
```suggestion
contains manifests which can be used to deploy kubenurse to the kube-system namespace of your cluster.
```

Up to discussion, not sure whether manifest is more standard than YAML or not 🤷🏻‍♂️